docs/mkdocs.yml

```diff
@@ -79,7 +79,7 @@ nav:
     - Examples:
       - user/examples/atm-fraud-pipeline.md
     - References:
-      - user/references/cli-commands.md
+      - CLI usage: user/references/cli-commands.md
```

Comment:
Because otherwise the title in the navigation bar would be the same as the first `# heading` in the file.

I added titles in the navigation, so that there is less implicit behavior. I think it is also more understandable, when you look at the `nav` section in `mkdocs.yaml` you can now see for sure what the titles of the pages are. Otherwise you have to visit each file.